### PR TITLE
Switching sandblast scripts dependencies

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Splitted scripts, also switched the usage of deprecated commands with none-deprecated ones,
and updated the docker image version.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by YuvHayun** [comment]
@ShahafBenYakir  will need for merge on this PR due to [CJ105] error.

File: Packs/Sandblast/ReleaseNotes/1_0_7.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+#### Scripts
+##### New: SbQuota
+- Updated the Docker image to: *demisto/python:2.7.18.27799*.
+- Fixed an issue where the script used ***sb-quota*** which is deprecated, the script is now using ***sandblast-quota*** instead.
```
Comment (ShahafBenYakir) [inline Packs/Sandblast/ReleaseNotes/1_0_7.md:5]: Let's fix for all scripts
```suggestion
- Fixed an issue where the script had used the deprecated command - ***sb-quota*** it is now using ***sandblast-quota*** instead.
```

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by YuvHayun** [comment]
@ShahafBenYakir  need force merge here

File: Packs/Sandblast/ReleaseNotes/1_0_7.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+#### Scripts
+##### New: SbQuota
```
Comment (ShahafBenYakir) [inline Packs/Sandblast/ReleaseNotes/1_0_7.md:3]: This is not new. Fix for all scripts.
```suggestion
##### SbQuota
```

**Comment by YuvHayun** [comment]
@ShahafBenYakir will need force merge due to [CJ105] Error.

**Pull request merged**